packages/fxa-auth-server/scripts/recorded-future/check-and-reset.ts

```diff
@@ -156,13 +161,52 @@ Limit: ${resultsPerPageLimit}
     }
   }
 
-  if (accountEmails.length === 0) {
+  if (accountLogins.length === 0) {
     console.log('No account emails found.');
     return 0;
   }
 
+  const credentialsLookupFn = createCredentialsLookupFn(client);
+  const cleartextCredentials = await credentialsLookupFn(accountLogins, {
+    first_downloaded_gte: firstDownloadedDateIsoString,
+  });
+  statsd.increment(
+    'recorded-future.identity.credentials-lookup.cleartext-total',
+    cleartextCredentials.length
+  );
+
+  const verifyPassword = createVerifyPasswordFn(
+    Password,
+    authDb.checkPassword.bind(authDb)
+  );
+
+  const defaultConcurrency = 100;
+  const queue = new PQueue({
+    concurrency: defaultConcurrency,
+    interval: 1000,
+    intervalCap: defaultConcurrency,
```

Comment:
Perhaps these should be configured individually? Is there a reason you'd want the concurrency to always equal the intervalCap?

Reply:
There is no good reason.  What would you default to here?  The `intervalCap` a multiple of `concurrency`?  Something else?

I'm want to keep this script ~dumb~simple.  And I feel picking a conservative upper bound is Good Enough.